### PR TITLE
fix #424

### LINKED ISSUE #424: Fix: generate_password_hash function default method type
Change `self._password = generate_password_hash(plaintext, method="sha256"))` to `self._password = generate_password_hash(password)` here https://github.com/Abdur-rahmaanJ/shopyo/blob/dev/shopyo/modules/box__default/auth/models.py#L99. Add a comment to mention that default hashing method is `pbkdf2:sha256`

### PR DESCRIPTION
**Pull request by akshitadixit**
This pull request closes #424

### CODE REVIEW

**Comment by rehmanis** [comment]
@akshitadixit Sorry I had made a mistake in the description. It should be `self._password = generate_password_hash(password)` (i.e just remove the `method` key word in the `generate_password_hash` everything else should be same). You can check locally first to see if tests are passsing. See step 6 of https://www.appinv.science/shopyo/contrib.html#making-a-pull-request. Also check step 8 as well to help remove the python limiting error mentioned in pep8speaks comment above.

File: shopyo/modules/box__default/auth/models.py [hunk 1]
```diff
@@ -96,7 +96,8 @@ def password(self):
 
     @password.setter
     def password(self, plaintext):
-        self._password = generate_password_hash(plaintext, method="sha256")
+        self.password = generate_password_hash(password)
+	#the default hashing method is pbkdf2:sha256
```
Comment (rehmanis) [inline shopyo/modules/box__default/auth/models.py]: put the comment above the function call:
```python
# The default hashing method is pbkdf2:sha256
self._password = generate_password_hash(password)
```

**Comment by akshitadixit** [comment]
Thank you for clarifying, I was skeptical if I did something wrong. Will fix.

**Comment by Abdur-rahmaanJ** [comment]
Dont worry about failing tests, i just modified workflow for next PR #429

**Comment by rehmanis** [comment]
> Dont worry about failing tests, i just modified workflow for next PR #429

It is failing for different reason.

**Comment by Abdur-rahmaanJ** [comment]
Yups for now

**Pull request closed without merging**